pom.xml

```diff
@@ -146,7 +146,12 @@
 			<dependency>
 				<groupId>org.slf4j</groupId>
 				<artifactId>slf4j-api</artifactId>
-				<version>1.7.8</version>
+				<version>1.7.10</version>
+			</dependency>
+			<dependency>
+				<groupId>org.slf4j</groupId>
+				<artifactId>slf4j-simple</artifactId>
+				<version>1.7.10</version>
```

Comment:
On a second thought, slf4j-simple is an exception here - it's hardly ever used outside of testing, but I'd still leave it as is, simply, for clarity. You decide :)